docs/docs/guides/core-types/evaluations.md

```diff
@@ -106,6 +106,25 @@ asyncio.run(evaluation.evaluate(model))
 
 This will run `predict` on each example and score the output with each scoring functions.
 
+#### Custom Naming
+
+You can change the name of the Evaluation itself by passing a `name` parameter to the `Evaluation` class.
+
+```python
+evaluation = Evaluation(
+    dataset=examples, scorers=[match_score1], name="My Evaluation"
+)
+```
+
+You can also change the name of individual evaluations by setting the `display_name` key of the `_weave_` dictionary.
+
+```python
+evaluation = Evaluation(
+    dataset=examples, scorers=[match_score1]
+)
+evaluation.evaluate(model, _weave_={"display_name": "My Evaluation Run"})
```

Comment:
by convention, use leading `_` or double underscore `__`

I think this is better named as `__weave_kwargs` or `__weave_settings`

Reply:
ack. changed to `__weave`